README.md

```diff
@@ -196,171 +196,12 @@ export function a11yAuditIf(contextSelector, axeOptions) {
-
-![](docs/assets/violation-log-output.png)
-
-##### Special Styling for Background-replaced elements
-
-As mentioned, some HTML elements are considered to be
-["replaced" elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Replaced_element), wherein their rendered appearance is -- to varying degrees --
-determined by the browser and/or the content that the element
-consists of.
-
-An `img` is an example of a replaced element; it lacks the ability to be
-styled through `background` definitions, as it's background is replaced with
-its loaded source assets.
-
-To get around this, Ember A11y Testing tries to detect these elements and --
-unless custom class names have been specified -- apply special styling independent
-of the component's current noise level:
-
-![](docs/assets/violation-replaced-bg-element.png)
-
-#### Component Hooks
-
-Since development is not a uniform experience, Ember A11y Testing provides
-several hooks to help stay out of the way.
-
-_Note:_ these are all `undefined` by default.
-
-##### Defining a custom callback
-
-If you feel the logging of violations is poor or you just want to see the entire
-results of a component's audit, you can define a custom callback. The callback
-receives the results of the `a11yCheck` audit that is scoped to the component's
-element. Simply set it as `axeCallback` on the component in question:
-
-```javascript
-axeCallback(results) {
-  // do stuff with results
-}
-```
-
-##### Setting options for the audit
-
-As with testing, if you need to set custom auditing options for a component, you
-can do so easily. Simply set a value for the `axeOptions` property value:
-
-```javascript
-axeOptions: {
-  /* a11yCheck options */
-}
-```
-
-##### Turning the audit off
-
-Lastly, if you really find the audits to be cramping development, you can turn
-them off via a simple boolean switch:
-
-```javascript
-turnAuditOff: true;
-```
-
-#### Environment Options
-
-Each of the fine-grained component hooks above can instead be defined for ALL components inside of your application's `config/environment.js` file. Simply supply them in a `componentOptions` hash on the `ember-a11y-testing` property of `ENV`.
-
-```javascript
-ENV['ember-a11y-testing'] = {
-  componentOptions: {
-    turnAuditOff: false, // Change to true to disable the audit in development
-    axeCallback: defaultAxeCallback,
-    axeOptions: defaultAxeOptions,
-    visualNoiseLevel: 2,
-    axeViolationClassNames: ['alert-box', 'alert-box--a11y'],
-  },
-};
-```
-
-By example, to enable a specific rule and to set options for a specific check, you can:
-
-```javascript
-ENV['ember-a11y-testing']: {
-  componentOptions: {
-    axeOptions: {
-      rules: {
-        'region': { enabled: true } // enable region rule
-      },
-      checks: {
-        'color-contrast': { options: { noScroll: true } } // disable scrolling of color-contrast check
-      }
-    }
```

Comment:
We should provide instructions on how to do this sort of thing in `test-helper.js` (assuming that's where it would go).

Reply:
Ya, it would basically be:

```js
// tests/test-helper.js

// ...snip...
import { setRunOptions } from 'ember-a11y-testing';

setRunOptions({
  rules: {
    region: { enabled: true },
  },
  checks: {
    'color-contrast': {
      options: {
        noScroll: true,
      },
    },
  },
});
```

Reply:
Ah yes. Agree. This was weird as there was an odd interleaving of the documentation for the test-based code and the component-based code. I would have chosen to keep them very separate, but oh well.

I can update the README with this example that @rwjblue showed.